packages/peregrine/lib/talons/CheckoutPage/PaymentInformation/usePaymentInformation.js

```diff
@@ -49,8 +47,7 @@ export const usePaymentInformation = props => {
      */
 
     const [doneEditing, setDoneEditing] = useState(false);
-    const [{ drawer }, { toggleDrawer, closeDrawer }] = useAppContext();
-    const isEditModalActive = drawer === 'edit.payment';
+    const [isUpdateMode, setIsUpdateMode] = useState(false);
```

Comment:
This naming looks familiar, but I don't think it applies to this state. Consider `isEditModalActive` or `isEditModalOpen` instead.